go.sum

```diff
@@ -89,6 +89,7 @@ github.com/hashicorp/golang-lru v0.5.3 h1:YPkqC67at8FYaadspW/6uE0COsBxS2656RLEr8
 github.com/hashicorp/golang-lru v0.5.3/go.mod h1:iADmTwqILo4mZ8BN3D2Q6+9jd8WM5uGBxy+E8yxSoD4=
 github.com/hpcloud/tail v1.0.0 h1:nfCOvKYfkgYP8hkirhJocXT2+zOD8yUNjXaWfTlyFKI=
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
+github.com/imdario/mergo v0.3.5 h1:JboBksRwiiAJWvIYJVo46AfV+IAIKZpfrSzVKj42R4Q=
```

Comment:
yes, i would like to separate it with two commits.
and i tried but i could not pass all checks during the build process since it requires some imports.  is there any suggestion? maybe the project does not use the vendor ?

Reply:
Sorry missed your comment.

You should be able to put the vendor commit first then the functional commit.

Reply:
i have separated two commits into vendor part and functional part, i moved functional part to #110 which currently can not pass all the checks, after this vendor part is done, i will fix it to pass all the checks.

Reply:
@nickngch Hi Nick, you can keep both commits in this PR. Sorry if I wasn't clear before. There is no need to separate them into separate PRs. We just need two Git commits.

Reply:
@yasker ok, it is done and  i closed #110